### ISSUE TITLE
OAuth2 avoid multiple confirmations when logging in multiple times

### ISSUE THREAD

**Issue by BigMichi1** [andryyy/mailcow-dockerized#4478]
## Summary

Provide an option for an OAuth2 app to suppress the confirmation screen (or follow up confirmations for some days/forever) when an external app is performing a login in via oauth. what is quite useful the first time is annoying after some days especially when you are logging in multiple times per day

## Motivation

What are you about to solve or improve with this idea?
What would be the benefit for most users?
user will not be prompted again and again to authorize the login

## Additional context

add the Trusted checkbox to the current screen for an OAuth app similar to the one from GitLab
current screen in MailCow
![Bildschirmfoto vom 2022-02-18 16-15-11](https://user-images.githubusercontent.com/91786/154710186-c58ffc58-ef80-4df3-938e-41192127f5b6.png)

Example from GitLab where only the first login needs to be confirmed because of the Trusted checkbox
![Bildschirmfoto vom 2022-02-18 16-16-16](https://user-images.githubusercontent.com/91786/154710179-10600ea2-e7ac-40c0-b4af-d7feb28318ef.png)

**Comment by waschbaer77** [comment]
We have the same "problem". It's annoying. This enhancement would help a lot.

**Comment by TaylorChristie** [comment]
The normal Oauth2 flow should require authorization for the first installation, then afterwards no longer request it. Having the user authorize every time is annoying and breaks the continuity of our integrations. +1 for this issue.

**Comment by Gri89** [comment]
I confirm, it would be very convenient.
I have authorization set up in several services, and to enter each one, login confirmation is required every time, which does not take much time, but is very annoying.

**Comment by mfuryto** [comment]
I agree. This is an important enhancement when running several services and users

**Comment by Fly7113** [comment]
I too second this enhancement. An option to configure this behaviour is mandatory.

**Comment by lauff** [comment]
Wondering that this obvious requirement is still open for more than two years. Unfortunately, I am not so familiar with GitHub processes. Is anyone working on this?

Anyhow, thanks for the great work!

Warm regards, Markus

**Comment by mfuryto** [comment]
I love all the good work you guys are doing. I wish I had the skills to contribute with this myself. Im doing a bump on this enhancement to get new attention to it because I know this would be a great fix for all mailcow admins not using an external IDP.

I think there could be two solutions:
1: There should be stored when the user click authorize, and then the flow would check before opening the authorize screen if the authorization is stored. If stored it bypass the authorization screen.
2: Trusted checkbox when adding oauth Client. Not showing the authorize screen at all. This is like the thread starter @BigMichi1 mentioned.
